playwright-e2e/tests/singular/enrollment/child-non-assenting-enrollement.spec.ts

```diff
@@ -0,0 +1,353 @@
+    await childSurveyPage.heightInFeet().fill('3');
+    await childSurveyPage.weightInPounds().fill('80');
+    await childSurveyPage.listedComplicationRelatedToHeartDisease().check('Liver Cancer (hepatocellular carcinoma)');
+    await childSurveyPage.listedComplicationRelatedToHeartDisease().check('Cardiac Arrest');
+    await childSurveyPage.hadArrhythmiaComplications().check('Did not require medical treatment');
+    await childSurveyPage.onEcmo().check('No');
+    await childSurveyPage.hadVad().check('No');
+    await childSurveyPage.hasReceivedHeartTransplant().check('No');
+    await childSurveyPage.hasReceivedLiverTransplant().check('No');
+    await childSurveyPage
+      .aboutYourChildHealth()
+      .check('My child had at least 5 sick visits to the doctor (not routine check-ups)');
+    await childSurveyPage
+      .aboutYourChildHealth()
+      .check('My child has missed 7 days or more from work or school due to illness');
+    await childSurveyPage.describePhysicalHealth().check('Somewhat healthy');
+    await childSurveyPage.familyDescribePhysicalHealth().check('Somewhat healthy');
+    await childSurveyPage.hasAnyConditions().check('Eating disorder');
+    await childSurveyPage.hasAnyConditions().check('Depression');
+    await childSurveyPage.hadNeurodevelopmentalNeurocognitiveEvaluation().check('Yes');
+    await childSurveyPage.hasDiagnosedWithAnyFollowings().check("I don't know");
+    await childSurveyPage.hasReceivedEducationSupportThroughSchool().check('IEP');
+    await childSurveyPage
+      .hasReceivedSupportOrTreatmentForBehavioralNeurodevelopmentalPsychologicalProblem()
+      .check('Yes');
+    await childSurveyPage.submit();
+
+    // Assert contents in My Dashboard table
+    await myDashboardPage.waitForReady();
+    const orderedHeaders = ['Title', 'Summary', 'Status', 'Action'];
+    const table = myDashboardPage.getDashboardTable();
+    const headers = await table.getColumnNames();
+    expect(headers).toHaveLength(4); // Four columns in table
+    expect(headers).toEqual(orderedHeaders);
+
+    const summaryCell = await table.findCellLocator('Title', 'Consent Form for Minor Dependent', 'Summary');
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    await expect(summaryCell!).toHaveText('Thank you for signing the consent form -- welcome to Project Singular!');
+
+    const statusCell = await table.findCellLocator('Title', 'Consent Form for Minor Dependent', 'Status');
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    await expect(statusCell!).toHaveText('Complete');
+
+    await expect(page.locator('.enrollmentStatusCompleteText')).toHaveText('Fully Enrolled');
+  });
+
+  test('enrolling non assenting child > 7 not capable of consent @enrollment @singular', async ({ page }) => {
+    // Assertion helper functions
+    const assertActivityHeader = async (page: Page, expectedText: string | RegExp) => {
```

Comment:
seems like a duplication of lines 27-33. 
I suggest extracting it like [here  ](https://github.com/broadinstitute/ddp-angular/pull/1830/files#diff-30eae4db7fabeff5de10196b1e171cd10752e48c860ee8cfc03a3c6e4ce9e09e:~:text=%40%40%20%2D0%2C0%20%2B1%2C9%20%40%40-,import%20%7B%20expect%2C%20Page%20%7D%20from%20%27%40playwright/test%27%3B,%7D%3B,-Footer).

Reply:
done